ui/analyse/src/study/studyChapters.ts

```diff
@@ -74,6 +74,9 @@ export function resultOf(tags: TagArray[], isWhite: boolean): string | undefined
   }
 }
 
+let filteredChapters = [];
+let filterChInputValue = '';
+
```

Comment:
These probably shouldn't be globals. They can be locals inside `view()`. And ofc, storing both shouldn't be necessary in the first place. Just the list should be enough.

Reply:
setting these inside view reinitializes them on any input event, as view is redrawn on each change.